lib/3scale/backend/transactor/usage_report.rb

```diff
@@ -57,16 +57,12 @@ def remaining_time(from = Time.now)
           # is predicted, not to be reported. The number of identical calls
           # that could be performed is 10/2 = 5.
           #
-          # Returns 0 when not authorized.
           # Returns -1 when there is not a limit in the number of calls.
           def remaining_same_calls
```

Comment:
I don't see this change being tested in the unit tests, just integration. Is that fine?

Reply:
I think it's fine. `UsageReport` does not have any unit tests. I guess it's because it was extracted from another class and its tests were not.

In this case, this method has only one caller, `LimitsHeader`, and I added a unit test for it. That unit test and the integration tests I added should cover everything introduced in this PR. Anyway, I'll consider adding one more for `UsageReport`.